doc/reference/storage_drivers.md

```diff
@@ -34,19 +34,20 @@ Optimized instance creation                 | no        | yes   | yes     | yes
 Optimized snapshot creation                 | no        | yes   | yes     | yes     | yes      | yes    | n/a         | yes
 Optimized image transfer                    | no        | yes   | no      | yes     | yes      | n/a    | n/a         | no
 {ref}`storage-optimized-volume-transfer`    | no        | yes   | no      | yes     | yes      | n/a    | n/a         | no
-{ref}`storage-optimized-volume-refresh`     | no        | yes   | yes[^1] | yes     | no       | n/a    | n/a         | no
+{ref}`storage-optimized-volume-refresh`     | no        | yes   | yes[^1] | yes     | yes[^2]  | n/a    | n/a         | no
 Copy on write                               | no        | yes   | yes     | yes     | yes      | yes    | n/a         | yes
 Block based                                 | no        | no    | yes     | no      | yes      | no     | n/a         | yes
 Instant cloning                             | no        | yes   | yes     | yes     | yes      | yes    | n/a         | no
-Storage driver usable inside a container    | yes       | yes   | no      | yes[^2] | no       | n/a    | n/a         | no
+Storage driver usable inside a container    | yes       | yes   | no      | yes[^3] | no       | n/a    | n/a         | no
 Restore from older snapshots (not latest)   | yes       | yes   | yes     | no      | yes      | yes    | n/a         | yes
-Storage quotas                              | yes[^3]   | yes   | yes     | yes     | yes      | yes    | yes         | yes
+Storage quotas                              | yes[^4]   | yes   | yes     | yes     | yes      | yes    | yes         | yes
 Available on `lxd init`                     | yes       | yes   | yes     | yes     | yes      | no     | no          | no
 Object storage                              | yes       | yes   | yes     | yes     | no       | no     | yes         | no
 
 [^1]: Requires [`lvm.use_thinpool`](storage-lvm-pool-config) to be enabled. Only when refreshing local volumes.
-[^2]: Requires [`zfs.delegate`](storage-zfs-vol-config) to be enabled.
-[^3]: % Include content from [storage_dir.md](storage_dir.md)
+[^2]: Only for volumes with content type `block`.
```

Comment:
Wait, I thought we solved this for filesystem ones too now in order to support the VM's config filesystem volume?

Reply:
You are right, that has to be changed, thanks!

Reply:
Fixed.